src/app/components/queues-view/queues-view.component.ts

```diff
@@ -226,4 +230,14 @@ export class QueuesViewComponent implements OnInit {
       });
     }
   }
+
+  loadStoredPartition(defaultValue = ''): string {
+    const storedPartition = localStorage.getItem('selectedPartitionAndQueue');
+    if (storedPartition && storedPartition.indexOf(':') > 0) return storedPartition.split(':')[0];
+    return defaultValue;
+  }
+
+  storeSelectedPartition() {
+    localStorage.setItem('selectedPartitionAndQueue', this.partitionSelected + ':');
+  }
```

Comment:
Is it possible to move below localStorage functions into same module?  or move to utils? 
If it's ok we can create another Jira to refactor it.

1. (app-view) loadStoredPartition(defaultValue = '')
2. (app-view) storeQueueSelection()
3. (node-view) loadStoredPartition(defaultValue = '')
4. (node-view) storeSelectedPartition() {
5. (queue-view) loadStoredPartition(defaultValue = '')
6. (queue-view) storeSelectedPartition() {

Reply:
It makes sense. I'm updating it with this ticket